cwms-data-api/src/main/java/cwms/cda/data/dto/TimeSeriesExtents.java

```diff
@@ -16,33 +39,27 @@
 @Schema(description = "TimeSeries extent information")
 @JsonPropertyOrder(alphabetic = true)
 @JsonNaming(PropertyNamingStrategies.KebabCaseStrategy.class)
-public class TimeSeriesExtents {
+public class TimeSeriesExtents extends TimeExtents {
 
     @Schema(description = "TimeSeries version to which this extent information applies")
     @JsonFormat(shape = Shape.STRING)
     ZonedDateTime versionTime;
 
-    @Schema(description = "Earliest value in the timeseries")
-    @JsonFormat(shape = Shape.STRING)
-    ZonedDateTime earliestTime;
-
-    @Schema(description = "Latest value in the timeseries")
-    @JsonFormat(shape = Shape.STRING)
-    ZonedDateTime latestTime;
-
     @Schema(description = "Last update in the timeseries")
     @JsonFormat(shape = Shape.STRING)
     ZonedDateTime lastUpdate;
 
     @SuppressWarnings("unused") // required so JAXB can initialize and marshal
     private TimeSeriesExtents() {
+        super(new Builder());
```

Comment:
This is just... odd.

The constructor should just take a builder and thus both constructors should be private. We appear to be mixing builder/non-builder patterns here and a rather odd way..

Reply:
I agree. I tried to preserve the existing TimeSeriesExtents object as much as possible, while making the new TimeExtents object follow the builder pattern of our other dtos. I would be in favor of updating the TimeSeriesExtents to use a builder pattern.

Reply:
I updated the TimeSeriesExtents to use a builder pattern for consistency